pkg/kademlia/kademlia.go

```diff
@@ -70,16 +70,20 @@ func NewKademlia(id dht.NodeID, bootstrapNodes []pb.Node, address string, identi
 	}
 	kdb, ndb := dbs[0], dbs[1]
 
-	rt, err := NewRoutingTable(&self, kdb, ndb, &RoutingOptions{
-		idLength:     kadconfig.DefaultIDLength,
-		bucketSize:   kadconfig.DefaultBucketSize,
-		rcBucketSize: kadconfig.DefaultReplacementCacheSize,
+	rt, err := NewRoutingTable(self, kdb, ndb, &RoutingOptions{
+		IDLength:     kadconfig.DefaultIDLength,
```

Comment:
Why are we exporting these variables? They should be changed by anything external.

Reply:
We need a way to create routing table with custom database outside from this package. (Starting an inmemory instance of the whole thing)

Reply:
I would also be fine with just having constants and not being able to customize these things. Or having a default routing options.